packages/terra-disclosure-manager/package.json

```diff
@@ -23,15 +23,10 @@
   "homepage": "https://github.com/cerner/terra-framework#readme",
   "peerDependencies": {
     "react": "^16.4.2",
-    "react-dom": "^16.4.2",
-    "terra-base": "^4.0.0"
```

Comment:
I removed the peer dependency on terra-base because the disclosure manager doesn't actually import anything from it.

Reply:
terra-doc-template requires a peer of terra-base. Should we remove it from this level?

Reply:
If terra-doc-template requires a peer dependency of terra-base, it'll still complain at install if one isn't provided. Removing it here will just result in one less package complaining about it missing, right?